src/AIStreaming/README.md

```diff
@@ -33,6 +33,26 @@ Endpoint=xxx;AuthType=azure
 
 And then you need to grant your user the `SignalR App Server ` role. For more connection string details, please access to [Connection String](https://learn.microsoft.com/en-us/azure/azure-signalr/concept-connection-string), and for more details about permission, please access to [Assign Azure roles for access rights](https://learn.microsoft.com/azure/azure-signalr/signalr-concept-authorize-azure-active-directory#assign-azure-roles-for-access-rights).
 
+In case you have Microsoft Defender for Cloud protection on your Azure OpenAI resource and you want to have additional user context on the alerts,
+please run:
+
+```bash
+azd env set MS_DEFENDERFORCLOUD_ENABLED true
+```
+
+To customize the application name of the user context, run this command:
+
+```bash
+azd env set APPLICATION_NAME <your application name>
+```
+
+and run again: 
+```bash
+azd up
+```
+
+For more details, refer to the [Microsoft Defender for Cloud documentation](https://learn.microsoft.com/azure/defender-for-cloud/gain-end-user-context-ai).
```

Comment:
Suggest moving this line to after the first line so the link will be close in case customer doesn't have it enabled yet